.github/workflows/ci.yaml

```diff
@@ -623,23 +623,26 @@ jobs:
         lint_markdown,
       ]
     runs-on: ubuntu-latest
-    if: |
-      always() &&
-      (needs.changed_files.result == 'success') &&
-      (needs.build_mirrord.result == 'success' || needs.build_mirrord.result == 'skipped') &&
-      (needs.test_agent_image.result == 'success' || needs.test_agent_image.result == 'skipped') &&
-      (needs.integration_tests_macos.result == 'success' || needs.integration_tests_macos.result == 'skipped') &&
-      (needs.integration_tests.result == 'success' || needs.integration_tests.result == 'skipped') &&
-      (needs.e2e.result == 'success' || needs.e2e.result == 'skipped') &&
-      (needs.test_mirrord_layer_cli.result == 'success' || needs.test_mirrord_layer_cli.result == 'skipped') &&
-      (needs.test_agent.result == 'success' || needs.test_agent.result == 'skipped') &&
-      (needs.lint.result == 'success' || needs.lint.result == 'skipped') &&
-      (needs.lint_macos.result == 'success' || needs.lint_macos.result == 'skipped') &&
-      (needs.test_mirrord_config.result == 'success' || needs.test_mirrord_config.result == 'skipped') &&
-      (needs.test_mirrord_protocol.result == 'success' || needs.test_mirrord_protocol.result == 'skipped') &&
-      (needs.test_mirrord_sip.result == 'success' || needs.test_mirrord_sip.result == 'skipped') &&
-      (needs.build_intellij_plugin.result == 'success' || needs.build_intellij_plugin.result == 'skipped') &&
-      (needs.lint_markdown.result == 'success' || needs.lint_markdown.result == 'skipped')
+    if: always()
```

Comment:
Maybe add a comment explaining why we need this?

Reply:
we don't and it actually might cause issues afaik